Gemfile

```diff
@@ -1,8 +1,12 @@
+# Remplazed frozen_string_literal true for something else
+
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.2.2'
 
+# gem 'sqlite3'
+
```

Comment:
![image](https://github.com/EmiLoKiJu/BlogApp/assets/93039359/7a4a1b46-de25-43e6-9d91-cbb1419df752)

- [ ] It would be nice if you rename your branch to `feature/add_validations_spec` to make your pull request professional 👍